### PR TITLE
Some Makefile cleanups

### PR DESCRIPTION
**Pull request by bgilbert**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,9 @@
-all:
-	go build .
 .PHONY: all
+all: fedora-coreos-stream-generator
+
+fedora-coreos-stream-generator: main.go go.mod Makefile
```
Comment (saqibali-2k) [inline Makefile:4]: In light of this change, does it still make sense to add a `go get -u` line in the `make` command? I.e we will update fedora-coreos-stream-generator if go.mod changes, but we are updating it to the latest commit (as opposed to the latest tagged commit in go.mod).

**Review by saqibali-2k — APPROVED** [review]
looks good :+1:

File: Makefile (hunk 1 above)
Comment (jlebon) [reply Makefile:4]: Isn't this missing all the other `.go` files though? This comment is on the same line as Saqib's comment, but unrelated.

Comment (jlebon) [reply Makefile:4]: > In light of this change, does it still make sense to add a `go get -u` line in the `make` command? I.e we will update fedora-coreos-stream-generator if go.mod changes, but we are updating it to the latest commit (as opposed to the latest tagged commit in go.mod).

It looks like `go get -u` matches the behaviour of Dependabot of sticking to tags if already on a tag. In that case, I think it'd make sense to have a `make update` target that runs `go get -u`. But I wouldn't make it part of the default build target.

Comment (bgilbert) [reply Makefile:4]: @jlebon Which other `.go` files?  There's only one in this repository.

Comment (jlebon) [reply Makefile:4]: Bahh sorry, was looking at the wrong tab.

**Review by jlebon — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (saqibali-2k) [reply Makefile:4]: > In that case, I think it'd make sense to have a make update target that runs go get -u.

good idea, will add this

**Pull request merged**